conf/app.routes

```diff
@@ -1,4 +1,4 @@
-GET        /movements/arrivals                                      controllers.ArrivalMovementController.getArrivalsForEori(updatedSince: Option[OffsetDateTime] ?= None)
+GET        /movements/arrivals                                      routing.ArrivalsRouter.getArrivalsForEori(updatedSince: Option[OffsetDateTime] ?= None)
```

Comment:
Again con not see any usage of updatedSince: Option[OffsetDateTime] in code

Reply:
as mentioned above, it will be dealt in separate ticket